cmd/csi-addons/replication.go

```diff
@@ -59,28 +60,65 @@ func (rep *VolumeReplicationBase) Init(c *command) error {
 		return errors.New("secret name is not set")
 	}
 
-	pv, err := getKubernetesClient().CoreV1().PersistentVolumes().Get(context.Background(), c.persistentVolume, metav1.GetOptions{})
-	if err != nil {
-		return fmt.Errorf("failed to get pv %q", c.persistentVolume)
-	}
-
-	if pv.Spec.CSI == nil {
-		return fmt.Errorf("pv %q is not a CSI volume", c.persistentVolume)
-	}
-
-	if pv.Spec.CSI.VolumeHandle == "" {
-		return errors.New("volume ID is not set")
+	if c.persistentVolume != "" {
```

Comment:
In case where both PV and VGRC are provided, PV will be considered.
error out before this if both are provided.